yoticlient.go

```diff
@@ -222,26 +220,21 @@ func getProtobufAttribute(profile Profile, key string) *yotiprotoattr.Attribute
 	return nil
 }
 
-func handleSuccessfulResponse(responseContent string, key *rsa.PrivateKey) (userProfile UserProfile, activityDetails ActivityDetails, errStrings []string) {
+func handleSuccessfulResponse(responseContent string, key *rsa.PrivateKey) (userProfile UserProfile, activityDetails ActivityDetails, err error) {
 	var parsedResponse = profileDO{}
-	var err error
 
 	if err = json.Unmarshal([]byte(responseContent), &parsedResponse); err != nil {
-		errStrings = append(errStrings, err.Error())
 		return
 	}
 
 	if parsedResponse.Receipt.SharingOutcome != "SUCCESS" {
 		err = ErrSharingFailure
```

Comment:
I couldn't actually find a code path where ErrStrings.length >1. Reasoning will be presented in comments...

ErrStrings was null at this point, so we don't need a list here.